### ISSUE TITLE
setQualityOfService: unrecognized selector

### ISSUE THREAD

**Issue by Dids** [LaunchKit/launchkit-ios#6]
Just pushed a new version of one of my apps using LaunchKit, but instantly started seeing crashes on Fabric/Crashlytics.

It's pointing at *[LKAPIClient init]* and crashing with:
```
Fatal Exception: NSInvalidArgumentException
-[NSOperationQueue setQualityOfService:]: unrecognized selector sent to instance 0x178232ba0
```

**Comment by rsattar** [comment]
Good catch Pauli, will have a fix very soon. Very sorry about that. Is this on iOS 7?

**Comment by Dids** [comment]
Ah, yes, seems to be iOS 7 only. That shouldn't be as big of an issue as I initially thought then. :)

**Comment by rsattar** [comment]
@Dids, fixed in 1.1.3, which I just pushed. Thanks for the find!

**Comment by Dids** [comment]
No problem. Loving the LaunchKit products. :)
